src/Entity/UserAgent.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Entity;
+
+class UserAgent
+{
+    /**
+     * @var string
+     */
+    private $ip;
```

Comment:
Some properties in this PR are typed, others are just documented with phpdoc. It makes sense to use the same standard.